spec/unit/multi_select_spec.rb

```diff
@@ -17,8 +17,12 @@ def output_helper(prompt, choices, active, selected, options = {})
     out = []
     out << "\e[?25l" if init
     out << prompt << " "
-    out << "(min. #{options[:min]}) " if options[:min]
-    out << "(max. #{options[:max]}) " if options[:max]
+    if options[:min] && options[:max]
+      out << "(min. #{options[:min]}, max. #{options[:max]}) "
+    else
+      out << "(min. #{options[:min]}) " if options[:min]
+      out << "(max. #{options[:max]}) " if options[:max]
+    end
```

Comment:
If we're optimizing for readability/ease of change this feels less optimal to me since the current version makes it clear that there are two distinct cases and the outcome is conspicuous for each case -- don't have to slow down to compute that mentally. But if happy to change if you feel strongly.